orchestra/models.py

```diff
@@ -10,7 +11,6 @@
 from orchestra.workflow import Step
 from orchestra.utils.assignment_snapshots import load_snapshots
 
-
 # TODO(marcua): Convert ManyToManyFields to django-hstore referencefields or
 # wait for django-postgres ArrayFields in Django 1.8.
```

Comment:
if we use arrayfields, we can push operations like filter/project into postgres even for JSON fields---the data is no longer opaque.  worth updating the comment?